apps/protocol-frontend/src/components/ContributionsByDateChart.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  Flex,
+  Box,
+  Divider,
+  Heading,
+  Text,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import { ResponsiveTimeRange, CalendarTooltipProps } from '@nivo/calendar';
+import { TooltipWrapper } from '@nivo/tooltip';
+import { GovrnTheme, GovrnSpinner } from '@govrn/protocol-ui';
+import pluralize from 'pluralize';
+import { TODAY_DATE, YEAR, BRAND_COLOR_MAP } from '../utils/constants';
+import { endOfDay, subWeeks, addDays } from 'date-fns';
+import { useContributionList } from '../hooks/useContributionList';
+
+type ContributionCount = {
+  date: string;
+  count: number;
+};
+interface ContributionsByDateChartProps {
+  contributionsCount: ContributionCount[] | undefined;
+  isFetching: boolean;
+  isLoading: boolean;
+  isError: boolean;
+  daoId: number;
+}
+
+const brandColors = GovrnTheme.colors.brand.primary;
+
+const ContributionsByDateChart = ({
+  daoId,
+  contributionsCount,
+  isFetching,
+  isLoading,
+  isError,
+}: ContributionsByDateChartProps) => {
+  const isMobile = useBreakpointValue({ base: true, lg: false });
+  const contributionsCountMap = contributionsCount?.map(contribution => {
+    return {
+      day: contribution.date,
+      value: contribution.count,
+    };
+  });
+
+  if (isError) {
+    return (
+      <Text>An error occurred fetching the DAO's recent Contributions.</Text>
+    );
+  }
+
+  if (isFetching || isLoading) {
+    return <GovrnSpinner />;
+  }
+
+  const CustomTooltipDailyContributions = ({
+    day,
+    value,
+  }: CalendarTooltipProps) => {
+    const { data: dailyContributions } = useContributionList({
+      first: 1000, // TODO: better way to do this -- do we want to have to use infinite list for this? we can also cap it at 10 or something
```

Comment:
Since this is in the tooltip I would say only show max 5 and order them by hightest attestation count. We probably shouldn't render 1000+ contributions in a tooltip

Reply:
Yeah for sure -- I was thinking 5 max is good -- definitely wasn't intending to render 1000. Probably 5 is a reasonable limit and then we can go from there